src/cpuAV.cpp

```diff
@@ -397,6 +415,46 @@ void CpuAndreVictor::showResponseOnDisplay(string value)
     }
 }
 
+float CpuAndreVictor::getCurrentValue(){
+    if(this->writeIndex){
+        return this->registerOne->getValue();
+    }
+    else(
+        return this->registerTwo->getValue();
+    )
+}
+
+void CpuAndreVictor::MPlus()
+{
+    float value = getCurrentValue();
+    float memoryValue = this->memoryRegister->getValue();
+    this->memoryRegister->setValue(to_string(value + memoryValue));
+}
+
+void CpuAndreVictor::MMinus()
+{
+    float value = getCurrentValue();
+    float memoryValue = this->memoryRegister->getValue();
+    this->memoryRegister->setValue(to_string(memoryValue - value));
+}
+
+void CpuAndreVictor::MRC()
+{
+    if (!this->memoryRegister->getIsDoublePress())
+    {
+        float response = this->memoryRegister->getValue();
+        stringstream stream;
+        stream << response;
+        string convertValue = stream.str();
+        showResponseOnDisplay(());
+        this->memoryRegister->setIsDoublePress(true);
+    }
+    else{
+        this->memoryRegister->reset();
+    }
+}
```

Comment:
Seguir padrão de nomenclatura.
```suggestion

void CpuAndreVictor::mPlus()
{
    float value = getCurrentValue();
    float memoryValue = this->memoryRegister->getValue();
    this->memoryRegister->setValue(to_string(value + memoryValue));
}

void CpuAndreVictor::mMinus()
{
    float value = getCurrentValue();
    float memoryValue = this->memoryRegister->getValue();
    this->memoryRegister->setValue(to_string(memoryValue - value));
}

void CpuAndreVictor::mrc()
{
    if (!this->memoryRegister->getIsDoublePress())
    {
        float response = this->memoryRegister->getValue();
        stringstream stream;
        stream << response;
        string convertedValue = stream.str();
        this->showResponseOnDisplay(convertedValue);
        this->memoryRegister->setIsDoublePress(true);
    }
    else{
        this->memoryRegister->reset();
    }
}
```